Mathlib/Topology/EMetricSpace/Paracompact.lean

```diff
@@ -1,7 +1,7 @@
 /-
-Copyright (c) 202 Yury G. Kudryashov. All rights reserved.
```

Comment:
I think the name might not be the only issue here...

Reply:
Yury is a pioneer of formalization.